addons/mail/static/src/components/attachment_box/attachment_box_tests.js

```diff
@@ -26,7 +25,7 @@ QUnit.module('attachment_box_tests.js', {
 
         this.createAttachmentBoxComponent = async (thread, otherProps) => {
             const props = Object.assign({ threadLocalId: thread.localId }, otherProps);
-            await createRootComponent(this, components.AttachmentBox, {
+            await createRootComponent(this, getMessagingComponent("AttachmentBox"), {
```

Comment:
I just noticed we could go even further actually, `createRootComponent` should just receive the name instead (and could be renamed to `createRootMessagingComponent` if the clarification is needed.

Then we can do the import and call of `getMessagingComponent` only once in the utils.